images/dptp-3312-hypershift-leaks-cleaner/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM amazon/aws-cli:latest
```

Comment:
If that is coming from dockerhub (as I think) we need to think about a different solution. It is rate-limited on our cluster. The image will not be downloaded.